packages/app/src/settings/device/NotificationSettings.vue

```diff
@@ -6,77 +6,104 @@
     <template #description>
       {{ t('settingsPage.notifications.description') }}
     </template>
-    <div class="divide-y border rounded divide-gray-50 border-gray-100 px-[16px]">
+    <div class="divide-y border rounded divide-gray-50 border-gray-100">
```

Comment:
These divider classes (`divide-y`, `divide-gray-50`) need to be moved down to the inner div now that there's another container div inside of this one. Right now it breaks the divider styling that separates each preference.

Reply:
Fixed in https://github.com/cypress-io/cypress/pull/26965/commits/36f826b1e3439d2662fe3631dcf52d08b055f824